src/k8s/pkg/utils/cidr.go

```diff
@@ -101,6 +103,59 @@ func ParseAddressString(address string, port int64) (string, error) {
 	return util.CanonicalNetworkAddress(address, port), nil
 }
 
+// GetDefaultAddress returns the default IPv4 and IPv6 addresses of the host.
+func GetDefaultAddress() (ipv4, ipv6 string, err error) {
```

Comment:
Is this intended to only be used in the tests?

Reply:
Yeah, I initially also used in the normal implementation but did not use it anymore. However, it simplifies the tests so I decided to keep it. (In utils as it might be useful in other context as well.)